app/models/food.rb

```diff
@@ -0,0 +1,26 @@
+# == Schema Information
+#
+# Table name: foods
+#
+#  id               :bigint           not null, primary key
+#  measurement_unit :string
+#  name             :string
+#  price            :decimal(, )
+#  quantity         :integer
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  user_id          :bigint           not null
+#
+# Indexes
+#
+#  index_foods_on_user_id  (user_id)
+#
+# Foreign Keys
+#
```

Comment:
- [ ] Kindly remove all the cmmented code in this file, this will make your code base to be neat. ℹ️ The same is applicable to `models/recipe.rb, models/recipe_food.rb and model/user.rb`